<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUEGenerator</class>
 <widget class="QMainWindow" name="CUEGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUEGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DIr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Artist</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="artistText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="albumText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>File Type</string>
    </property>
    <widget class="QRadioButton" name="flacButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FLAC</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="apeButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>APE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="wavButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>WAV</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="generate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GO!</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
